src/Core/SharedKernels/DataCollection/DataCollection.Portable/IRosterLevel.cs

```diff
@@ -5,10 +5,6 @@ namespace WB.Core.SharedKernels.DataCollection
     public interface IRosterLevel
     {
         decimal @rowcode { get; }
-        [Obsolete("obsolete since 5.10")]
-        string @rowname { get; }

```

Comment:
do we support old assembly or not?

Reply:
We agreed to move forward and do not have it any more